Themes with names consisting of two words don't work

The way `rsg-codemirror-theme.css` is generated, themes with names different from the `.css` files don't work. This error is raised: 

```
Module not found: Can't resolve 'rsg-codemirror-theme.css' in '/home/dhruvkb/Documents/gsoc/cc-vocabulary/node_modules/vue-styleguidist/lib/rsg-components/VsgEditor'
```

For example, the theme `solarized dark` (notice the absence of a dash) is loaded from the file `solarized.css`.

To reproduce, just set
```js
editorConfig: {
  theme: 'solarized dark'
}
```

To resolve this, as far as I know, the theme `.css` file should be generated using the word before the space in the theme name.

Since there is only one theme in the present CodeMirror collection that has this discrepancy (Solarized) so you might be inclined to dismiss this issue, but _please don't_.

--- a/examples/basic/styleguide.config.js
+++ b/examples/basic/styleguide.config.js
@@ -57,6 +57,9 @@ module.exports = {
 	},
 	usageMode: 'expand',
 	exampleMode: 'expand',
+	editorConfig: {
+		theme: 'solarized dark'
+	},
 	compilerConfig: {
 		target: { ie: 11 }
 	},

--- a/packages/vue-styleguidist/scripts/make-webpack-config.js
+++ b/packages/vue-styleguidist/scripts/make-webpack-config.js
@@ -40,7 +40,9 @@ module.exports = function(config, env) {
 		resolve: {
 			extensions: ['.js', '.jsx', '.json'],
 			alias: {
-				'rsg-codemirror-theme.css': `codemirror/theme/${config.editorConfig.theme}.${'css'}`
+				'rsg-codemirror-theme.css': `codemirror/theme/${
+					config.editorConfig.theme.split(' ')[0]
+				}.css`
 			}
 		},
 		module: {

